bin/check_goimports

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# Checks if all project files are formatted using goimports
+#
+# Usage:
+#> bin/check_goimports
+
+source bin/helpers/output.sh
+
+unformatted=`find . \
```

Comment:
this can be done simpler way: `goimports -e -l $(go list -f {{.Dir}} ./...)`

Reply:
Improved